src/openms/include/OpenMS/FORMAT/PepXMLFile.h

```diff
@@ -149,6 +149,12 @@ namespace OpenMS
     */
     void matchModification_(const double mass, const String& origin, String& modification_description);
 
+    /**
+        @brief create a TPP compatible string of the modified sequence using bracket notation
+        i.e.: n[35]RQLNK[162]LQHK[162]GEA
+    */  
+    static String modifiedSequenceToBracketString_(const AASequence & seq);
```

Comment:
I am not sure about putting this into `PepXMLFile`, shouldnt this go into `AASequence` or a helper class?